project/src/main/ui/chat/chat-choices.gd

```diff
@@ -72,6 +72,8 @@ func show_choices(choices: Array, moods: Array, new_columns: int = 0) -> void:
 	_refresh_child_buttons()
 	
 	if choices:
+		$EnableInputTimer.start()
```

Comment:
Maybe worth documenting that this timer actually disables input for a brief period of time

Reply:
Added code comment, `briefly disable input to prevent the player...`